sirepo/pkcli/admin.py

```diff
@@ -49,20 +49,20 @@ def _audit_user(uid, proprietary_sim_types):
                 )
 
     def _link_or_unlink_proprietary_files(sim_type, should_link):
-        d = proprietary_code_dir(sim_type)
-        for e in simulation_db.examples(sim_type):
-            b = sim_data.get_class(sim_type).proprietary_lib_file_basename(e)
-            p = simulation_db.simulation_lib_dir(sim_type).join(b)
-            if not should_link:
-                pkio.unchecked_remove(p)
-                continue
-            try:
-                p.mksymlinkto(
-                    d.join(b),
-                    absolute=False,
-                )
-            except py.error.EEXIST:
-                pass
+        b = sim_data.get_class(sim_type).FLASH_RPM_FILENAME
```

Comment:
I think the notion of FLASH_RPM_FILE_NAME should not be here. What this could should do is link `flash-*` (or `flash/*`) into the directory. It shouldn't know about flash specifically, just `sim_type`. It might be that you want to link a directory to the lib directory that is called something fixed like `proprietary_code` for all sim types.

Reply:
https://github.com/radiasoft/sirepo/pull/2490/commits/00903ca4af2b437c4de426a53cfab1f8b79638eb